model_cards/NlpHUST/vibert4news-base-cased/README.md

```diff
@@ -1,7 +1,3 @@
----
-language: vn
----
-
```

Comment:
I'm curious why you removed the language tag?

Reply:
sorry, I mistakenly deleted